public-ui/src/app/data-browser/cdr-version/cdr-version-info.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  Output,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import { environment } from 'environments/environment';
+import { Configuration, DataBrowserApi } from 'publicGenerated/fetch';
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import * as ReactDOM from 'react-dom';
+import { BaseReactWrapper } from '../../data-browser/base-react/base-react.wrapper';
+const api = new DataBrowserApi(new Configuration({ basePath: environment.publicApiUrl }));
+
+const containerElementName = 'root';
+
+export const style = `
+.result-body-item {
+    padding: 6px 0;
+}
+.cdr-info {
+    font-size: 14px;
+}
+`;
+
+interface State {
+    numParticipants: any;
+    creationTime: any;
```

Comment:
Nit: I'd recommend being as specific as you can when assigning types to state/props, and avoid generic `any`s whenever possible.
```suggestion
    numParticipants: string;
    creationTime: string;
```